src/main/java/com/fontys/api/service/TournamentService.java

```diff
@@ -80,11 +86,41 @@ public String deleteTournament(Integer id) {
         return Optional.empty();
     }
 
-    private void validateTournament(Integer id) throws InvalidAttributeValueException {
-        if (tournamentRepository.findById(id).isEmpty()) {
+    @Transactional
+    public String addMatchToTournament(Integer tournamentId, Integer matchId)
+    {
+        Optional<Tournament> tournament = tournamentRepository.findById(tournamentId);
+        Optional<Match> match = matchRepository.findById(matchId);
+
+        if(tournament.isEmpty())
```

Comment:
This should be done in a validator

Reply:
The code has been changed according to your feedback.

Reply:
Resolving this becuase there in a newer version